deployments/helm/nvidia-dra-driver-gpu/templates/controller.yaml

```diff
@@ -69,6 +69,11 @@ spec:
         # Use runc: explicit "void"; otherwise we inherit "all".
         - name: NVIDIA_VISIBLE_DEVICES
           value: void
+        {{- if .Values.featureGates }}
+        # Feature gates (includes both project-specific and logging features)
```

Comment:
Do you have a real-world example for a logging-related feature gate?

Reply:
They are built in to the logging abstraction we use. The three available that I knowing of are listed as the "examples" of what to set in the helm chart.